charged-ieee/typst.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "charged-ieee"
-version = "0.1.2"
-compiler = "0.10.0"
+version = "0.2.0"
+compiler = "0.11.1"
```

Comment:
Do any of these changes actually require 0.11.1?

Reply:
why the thumbs down?

Reply:
I meant to indicate that the changes don't appear to require the 0.11.1
Sorry for the ambiguity

Reply:
ah, I see!